app/renderer/lib/appium-client.js

```diff
@@ -1,9 +1,15 @@
 import _ from 'lodash';
 import Bluebird from 'bluebird';
-import {getWebviewStatusAddressBarHeight, parseSource, setHtmlElementAttributes} from './webview-helpers';
-import {SCREENSHOT_INTERACTION_MODE, APP_MODE} from '../components/Inspector/shared';
+import { parseSource, setHtmlElementAttributes } from './webview-helpers';
+import { SCREENSHOT_INTERACTION_MODE, APP_MODE } from '../components/Inspector/shared';
 
-const {TAP, SWIPE, GESTURE} = SCREENSHOT_INTERACTION_MODE;
+const { TAP, SWIPE, GESTURE } = SCREENSHOT_INTERACTION_MODE;
+
+// Selector for the Android webview - includes the correct top and bottom boundaries
+const ANDROID_WEBVIEW_SELECTOR = '//android.webkit.WebView[1]';
```

Comment:
I used xpath here mainly because I've seen apps that have more than one webview element (one parent, one child). Admittedly, UiA2 also does not detect any webview contexts in such a screen, but I decided to go the safe route.

Reply:
class name selectors are faster. Also it would always the first match - exactly what's needed here.
Btw, I am not sure about the correctness of this locator in general - it won't find the first WebView on the page, but rather each first child. Check https://stackoverflow.com/questions/52165349/what-is-the-role-of-parentheses-in-xpath-1-0 for more details.

Reply:
Fixed